internal/services/redisenterprise/redis_enterprise_database_resource.go

```diff
@@ -341,7 +341,6 @@ func resourceRedisEnterpriseDatabaseRead(d *pluginsdk.ResourceData, meta interfa
 	}
 
 	d.Set("name", id.DatabaseName)
-	d.Set("resource_group_name", id.ResourceGroupName)
```

Comment:
This probably needs to be wrapped in the beta flag for now and removed post-4.0 instead?

Reply:
Ooops! Missed this one. It's actually wrapped in that beta flag check just below this.